src/Persistence/PersistentProxyObjectFactory.php

```diff
@@ -42,23 +49,26 @@ abstract class PersistentProxyObjectFactory extends PersistentObjectFactory
     abstract public static function class(): string;
 
     /**
-     * @return T&Proxy<T>
+     * @return Proxy<T>
```

Comment:
Curiously enough, PHPStorm does not understand `T&Proxy<T>` but it does understand `list<T&Proxy<T>>` :shrug: